masterlist.yaml

```diff
@@ -112,9 +131,49 @@ globals:
         text: '[최신 LOOT 스레드](%1%).'
       - lang: zh_CN
         text: '[最新的LOOT版本](%1%)。'
+      - lang: da
+        text: '[Seneste LOOT-tråd](%1%).'
       - lang: ja
         text: '[LOOTの最新スレッドを開く](%1%)（英語）'
     subs: [ 'https://loot.github.io/latest-thread/' ]
+# SKSE Latest
+  - type: say
+    condition: 'file("../skse64_loader.exe") and version("../skse64_loader.exe", "0.2.0.6", <)'
```

Comment:
I didn't think of this before, but this condition can be shortened to just `version("../skse64_loader.exe", "0.2.0.6", <)`, if the file doesn't exist it will be false.

Reply:
I'll test it in a bit and push change.

Reply:
Tested &  
- using both != and < would always show skse latest version message.

Reply:
Ah, sorry, my mistake, `version()` will return true if the file doesn't exist for `!=`, `<` and `<=`, though thinking about it again I'm not sure what would be most intuitive.